samples/trivial/BUILD.bazel

```diff
@@ -2,8 +2,8 @@ load("@rules_sqldelight//:sqldelight.bzl", "sqldelight_codegen")
 load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "define_kt_toolchain", "kt_android_library")
 
 kt_android_library(
-    name = "r1_4_0",
-    srcs = glob(["src/main/java/**/*.java"]) + [":r1_4_0_sqldelight"],
+    name = "library",
+    srcs = glob(["src/main/java/**/*.java"]) + [":generated_sources"],
     deps = [
         "@maven_sqldelight//androidx/sqlite",
```

Comment:
Would it make sense to make this consistent with the other sample project and use `"@maven//com/squareup/sqldelight:jdbc-driver"` as a dependency here?

Reply:
No, because the trivial sample is actually an android sample. It needs sqlite. The JDBC bit is different, for non-android. I believe this doesn't even use JDBC at all as an API, but directly interacts with android's sqlite APIs.